Source/Core/EllipseGeometry.js

```diff
@@ -175,15 +175,15 @@ define([
                         }
                     }
 
-                    if (vertexFormat.binormal) {
-                        binormal = Cartesian3.normalize(Cartesian3.cross(normal, tangent, binormal), binormal);
-                        binormals[i] = binormal.x;
-                        binormals[i1] = binormal.y;
-                        binormals[i2] = binormal.z;
+                    if (vertexFormat.bitangent) {
+                        bitangent = Cartesian3.normalize(Cartesian3.cross(normal, tangent, bitangent), bitangent);
+                        bitangents[i ] = bitangent.x;
+                        bitangents[i1] = bitangent.y;
+                        bitangents[i2] = bitangent.z;
                         if (extrude) {
-                            binormals[i + bottomOffset] = binormal.x;
-                            binormals[i1 + bottomOffset] = binormal.y;
-                            binormals[i2 + bottomOffset] = binormal.z;
+                            bitangents[i  + bottomOffset] = bitangent.x;
```

Comment:
Whitespace. This occurs several other times, but other than that, I think this looks good.